runtime/kusama/src/gov_config.rs

```diff
@@ -100,18 +102,28 @@ pub mod pallet_custom_origins {
 		ReferendumCanceller,
 		/// Origin able to kill referenda.
 		ReferendumKiller,
+		/// Origin able to spend up to 1 KSM from the treasury at once.
+		SmallTipper,
```

Comment:
Does it make sense to have `Spender { max: Balance }`?

Reply:
Yes. That's done now.